src/Rings/groebner.jl

```diff
@@ -318,13 +318,13 @@ See [Fau99](@cite) for more information.
 
 # Examples
 ```jldoctest
-julia> R,(x,y,z) = polynomial_ring(GF(101), ["x","y","z"], ordering=:degrevlex)
+julia> R,(x,y,z) = polynomial_ring(GF(101), ["x","y","z"])
```

Comment:
@ederc I would like to remove the keyword here because it is part of the doc string which makes it look as if one needs to set the ordering like this. Can you confirm that it is not mathematically necessary to set the ordering?
Notice also the small change when printing the ideal below.

Reply:
Yes, you can remove the keyword here.